ocaml/testsuite/tests/typing-layouts-float64/basics_beta.ml

```diff
@@ -22,6 +22,11 @@
    convenient example of a concrete [float64] type in some tests, but its
    behavior isn't the primary purpose of this test. *)
 
+#directory "+stable"
+#load "stable.cma"
+[%%expect {|
+|}];;
```

Comment:
It would be nice if this was done by `include stable;`. Not sure how feasible that is, since ocamltest can be unwieldy.

Reply:
Done with https://github.com/ocaml-flambda/flambda-backend/pull/2558.